Source/WebDriver/WebSocketServer.cpp

```diff
@@ -82,14 +81,7 @@ RefPtr<Session> WebSocketServer::session(const WebSocketMessageHandler::Connecti
         }
     }
 
-    if (sessionId.isNull())
-        return { };
-
-    const auto& existingSession = m_service.session();
-    if (!existingSession || (existingSession->id() != sessionId))
-        return { };
-
-    return existingSession;
+    return sessionId;
```

Comment:
Instead of breaking in the loop we can just return pair.value and then we don't need the local variable, here we can just return { }